onnx_xla/operator_registry.cc

```diff
@@ -0,0 +1,51 @@
+#include "onnx_xla/operator_registry.h"
+#include <iostream>
+namespace onnx_xla  {
+  
+  OperatorRegistry::OperatorRegisterOnce::OperatorRegisterOnce(const Symbol& nodeKind, TranslationFunction translator)  {
+    auto& map = OperatorRegistry::map();
+    if (map.find(nodeKind) == map.end())  {
+      map[nodeKind] = translator;
+    } else {
+      throw std::runtime_error("Operator registry error");
+    }
+  }
+
+  onnxStatus OperatorRegistry::executeTranslation(Node& n, XlaBuilder& builder, ValueOpMap& valueToOp)  {
```

Comment:
nit: maybe call it translate instead of executeTranslation?

Reply:
sure, more concise -changed